packages/core/docs/integrate/integration-guide.md

```diff
@@ -5,319 +5,232 @@ The application does not reload automatically after saving the changes in Server
+To make it easy to get started, we created an [eCommerce integration boilerplate](https://github.com/vuestorefront/ecommerce-integration-boilerplate).
 
-## Scope
+It's a monorepo, which is a single repository containing multiple related projects. Each directory inside `packages` contains one project. There are three projects:
+
+- `api-client`,
+- `composables`,
+- `theme`.
 
-Vue Storefront will require **at least** the following features from your eCommerce platform:
+### API client
 
-**Product**
+`api-client` is the **_server layer_** that extends our [Server Middleware](../advanced/server-middleware.html). It creates an API client (like `Apollo` for GraphQL or `Axios` for plain HTTP) that communicates with your eCommerce platform. It acts as a proxy between the users and the platform.
 
-- fetch single/multiple products based on uuid/sku/catId/query
-- fetch product variants
+Here, you will create new endpoints that accept parameters sent from the frontend and use them to fetch or submit data to the platform.
 
-**Category**
+### Composables
 
-- fetch single category
-- fetch category tree
+`composables` consists of two parts:
 
-**Cart**
+- [Composables](../guide/composables.html) manage the state, prepare and send the request to the `api-client`, then save the response. If necessary, they also modify the response to simplify getters.
 
-- create cart
-- add to cart
-- remove from the cart
-- change the quantity of an item in the cart
-- change the configuration of the item in the cart
-- clear cart
-- add promo coupon
-- remove promo coupon
+- [Getters](../guide/getters.html) extract data from API responses provided by `composables` and return them in formatted and agnostic form.
 
-**User**
+Here, you will create new methods for `composables` to fetch the data and new `getters` to extract different pieces of information from that data.
 
-- log in
-- log out
-- register
-- forgot password
-- add/remove shipping address
-- set the default shipping address
-- add/remove billing address
-- set the default billing address
-- see past orders
+### Theme
 
-**Wishlist**
+This project is a template for generating new Vue Storefront shops. It's a [Nuxt.js](https://nuxtjs.org/) application that contains pages, Vue components, and assets. It uses `composables` to interact with the platform and `getters` to display the data to the user. 
 
-- add/remove the product
-- associate with the logged-in user
+Out of the box, the `theme` directory doesn't contain much - just a few configuration files and empty directories. However, this doesn't mean that you have to create the whole theme from scratch. When your integration is ready, you can use our CLI to combine this project with our base theme to generate a new Nuxt.js application with all necessary pieces inside.
```

Comment:
Actually, one of these `directories` could stay a `folder` :) The last comment intention was to avoid using same word in the same sentence:

```suggestion
Out of the box, the `theme` directory doesn't contain much - just a few configuration files and empty folders. However, ...
```